### PR TITLE
git commit

### PR DESCRIPTION
**Pull request by Gosha04**

### CODE REVIEW

**Comment by Gosha04** [comment]
Questions were changed

**Comment by spencerau** [comment]
@Gosha04 can you sync with the dev-fall2024 upstream and then edit your questions.json file so that it has the hexcode colors for each answer?

https://github.com/Learn-Adapt-Build/DiscoBall-App/blob/dev-fall2024/web-app/public/questions.json

**Comment by Gosha04** [comment]
I don't have a clue what colors each answer should have, I wasn't really briefed on that. Is there a doc with all the hexcodes?

________________________________

**Comment by spencerau** [comment]
Yes the document linked at questions.json has the hexcodes

**Comment by Gosha04** [comment]
Added
________________________________

File: web-app/public/questions.json
Comment (spencerau) [inline web-app/public/questions.json:1]: Why does this only have 14 questions?

**Review by spencerau — CHANGES_REQUESTED** [review]

File: web-app/public/questions.json
Comment (spencerau) [inline web-app/public/questions.json:1]: Why does this only have 14 questions?

File: web-app/public/questions_db.json
Comment (spencerau) [inline web-app/public/questions_db.json:1]: Can you delete this and/or update your fork to the upstream, so sync it with LAB:dev-fall2024

**Review by spencerau — CHANGES_REQUESTED** [review]
See comments

File: web-app/public/questions.json
Comment (Gosha04) [reply web-app/public/questions.json:1]: Not sure. Maybe deleted some in while adding colors? I'll look it over

**Pull request closed without merging**